kitsune/coolsearch/static/coolsearch/js/actions/ResultsActionCreator.js

```diff
@@ -0,0 +1,14 @@
+import SearchDispatcher from '../dispatcher/SearchDispatcher.js';
+import ActionConstants from '../constants/ActionConstants.js';
+
+
+var ResultsActionCreator = {
+    createResults: function (results) {
+        SearchDispatcher.dispatch({
+            type: ActionConstants.RECEIVE_RESULTS,
+            results: results
+        });
+    }
+};
+
+export default ResultsActionCreator;
```

Comment:
I've been preferring a style like

```js
export function foo() {}
export function bar() {}
export functino baz() {}

export default {
  foo,
  bar,
  baz,
};
```

This makes each individual part exportable with something like `import {foo} from './utils.js';`, while also keeping the `import utils from './utils.js';` ability. It also tends to be a bit less typing, and one level less indentation.